.talismanrc

```diff
@@ -1,6 +1,6 @@
 fileignoreconfig:
 - filename: packages/mds-db/sql-utils.ts
-  checksum: c5448443595c520b55184bfb67a4b4ee0f346d6b1864d8738268c52177c567a8
+  checksum: 532fb512aa6868689a76d3a49904c56e255b77c98abbe7e1b57e640ef344d4e6
```

Comment:
What's the story with this checksum?

Reply:
`talisman` is part of our `pre-commit` git hook that attempts to prevent the possibility of commiting secrets. Personally, I don't care for the way it works. It has the advantage of being installed as a dev dependency, but I'd love to find a better alternative.